src/components/badge/badge-test.stories.tsx

```diff
@@ -33,9 +33,28 @@ export const DefaultStory = ({ counter, ...args }: BadgeStoryProps) => {
   );
 };
 
+export const DisplayOnlyStory = ({ counter, ...args }: BadgeStoryProps) => {
+  return (
+    <div style={{ margin: "40px" }}>
+      <Badge counter={counter} {...args}>
+        <Button mr={0} buttonType="tertiary">
+          Filter
+        </Button>
+      </Badge>
+    </div>
+  );
+};
+
 DefaultStory.story = {
   name: "default",
   args: {
     counter: 1,
   },
 };
+
+DisplayOnlyStory.story = {
```

Comment:
This syntax is already deprecated, might be worth to refactor it like this:

DisplayOnlyStory.storyName = ...
DisplayOnlyStory.args = { ... }

I'd do the same for DefaultStory too

Reply:
Completely forgot about this too. I just coped as was in the file already. I'll get this changed too 👍🏻